src/EnsembleKalmanInversion.jl

```diff
@@ -63,13 +144,10 @@ function update_ensemble!(
 
     # u: N_par × N_ens 
     # g: N_obs × N_ens
-    u = get_u_final(ekp)
+    u = deepcopy(get_u_final(ekp))
```

Comment:
https://github.com/CliMA/EnsembleKalmanProcesses.jl/blob/8c3be1d253cef55df27ec1e88905d6a8d226f76a/src/DataContainers.jl#L96

All get methods with a `DataContainer` object call a deepcopy already so this is not required